src/advanced_inv.cpp

```diff
@@ -107,8 +107,6 @@ advanced_inventory::~advanced_inventory()
         werase( mm_border );
         werase( left_window );
         werase( right_window );
-    }
-    if( exit ) {
```

Comment:
I'm not sure about this one here - it could be `!exit` or something.

Reply:
Technically `exit` (being a member) could have been changed from external code, but the calls to `werase` certainly don't do this, so it will have the same value as in the `if` condition above. The second check for it is indeed not needed.

Reply:
Thanks!